package.json

```diff
@@ -17,6 +17,7 @@
     "yup": "^0.32.11"
   },
   "scripts": {
+    "analyze": "npm run build && source-map-explorer 'build/static/js/*.js'",
```

Comment:
I think this should be `react-scripts build && source-map-explorer 'build/static/js/*.js`. No need to run `makefsdata.js` since it's not part of the analysis.

Reply:
Update to this. We're not flagged for generating source maps for production builds, so we'll need a way to set `GENERATE_SOURCEMAP=true` when running the analyzer. Updated analyze command:
`cross-env GENERATE_SOURCEMAP=true react-scripts build && source-map-explorer 'build/static/js/*.js`

Reply:
Ah right, of course!
Will update, thanks!